test/MUXControls.Test/Infra/TestHelpers.cs

```diff
@@ -38,9 +38,13 @@ public class TestSetupHelper : IDisposable
     {
         private bool AttemptRestartOnDispose { get; set; }
 
+        public TestSetupHelper(string testName, string languageOverride = "", bool attemptRestartOnDispose = true)
+            :this(new string[] { testName }, languageOverride, attemptRestartOnDispose)
+        {}
+
         // The value of 'testName' should match that which was used when
         // registering the test in TestInventory.cs in the test app project.
-        public TestSetupHelper(string testName, string languageOverride = "", bool attemptRestartOnDispose = true)
+        public TestSetupHelper(ICollection<string> testNames, string languageOverride = "", bool attemptRestartOnDispose = true)
```

Comment:
If nested TestSetupHelper isn't supported anymore can you set a bit on the current thread (or really static in the whole process should be fine) that a TestSetupHelper is already active so you can't create another one? I know you found two that were using it but I thought there were more usages like in Scroller and NavigationView.

Reply:
Yeah there are more usages that I haven't updated. Just wanted to get some feedback of this approach, if people are ok with it I'll go ahead and update the rest. I'll try making it static too, that sounds like a good idea : )

Reply:
Oh cool. Yeah I think this approach is fine. That said, can TestHelper just know if there's another TestHelper active to make the existing pattern "just work"? Or do you think the new pattern is cleaner? I could believe the new pattern is cleaner and we should switch over to it wholesale, i just haven't thought through it.

Reply:
I think the new pattern is cleaner and we should just update the existing code to the new pattern. There shouldn't be too many places that need to be updated. And we should add a static bool to ensure that future tests don't get inadvertently written the old way.

Reply:
Yeah, we'll have to add some global states to track the opened TestHelper to make current pattern "just work". I think the tricky part is to correctly update the states when test run encounter exceptions/finishes. This new pattern should be less error prone.